lib/codecs/src/decoding/format/gelf.rs

```diff
@@ -21,10 +21,10 @@ use crate::gelf::GELF_TARGET_PATHS;
 use crate::{gelf_fields::*, VALID_FIELD_REGEX};
 
 /// On GELF decoding behavior:
-///   Graylog has a relaxed decoding. They are much more lenient than the spec would
-///   suggest. We've elected to take a more strict approach to maintain backwards compatibility
-///   in the event that we need to change the behavior to be more relaxed, so that prior versions
-///   of vector will still work with the new relaxed decoding.
+///   Graylog has a relaxed decoding algorithm. It is much more lenient than the spec would
```

Comment:
I removed the use of "we" because it's unclear who "we" is and also removed "Vector", let me know if the rewording works.

Reply:
Reverted changes.